cnxdb/migrations/20170911201035_transform_cnxml_to_html.py

```diff
@@ -0,0 +1,114 @@
+# -*- coding: utf-8 -*-
+import logging
+import time
+from datetime import timedelta
+
+from dbmigrator import deferred
+import rhaptos.cnxmlutils
+
+logger = logging.getLogger('dbmigrator')
```

Comment:
We should probably be doing `from dbmigrator import logger`. But this works!

Reply:
Yeah I thought about that too.  Let me change it.